src/SimpleAuth.Droid/SimpleAuth.Droid.csproj

```diff
@@ -47,6 +48,9 @@
     <Reference Include="System.ComponentModel.DataAnnotations" />
     <Reference Include="System.Core" />
     <Reference Include="System.Net.Http" />
+    <Reference Include="System.Runtime.Serialization">
+      <HintPath>..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETCore\v4.5\System.Runtime.Serialization.dll</HintPath>
```

Comment:
Adding references like this does not work. It is trying to use a desktop dll, for mobile.